### ISSUE TITLE
Adding a manual trigger task

### ISSUE THREAD

**Issue by CecileRobertMichon** [Microsoft/azure-pipelines-tasks#9477]
## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

## Environment
- Server - Azure Pipelines or TFS on-premises? Azure Pipelines
        
    - If using Azure Pipelines, provide the account name, team project name, build definition name/build number: https://dev.azure.com/aks-engine/AKS-Engine

- Agent - Hosted or Private: Hosted
    
    - If using Hosted agent, provide agent queue name: Ubuntu 16.04

## Issue Description

My team is migrating from circle CI to Azure DevOps and so far our experience has been good. However, there is one thing we were able to do with circle that we don't know how to do with pipelines and that is how to add a manual trigger for PR E2E build job. This is the previous definition we had for circle:
https://github.com/Azure/aks-engine/blob/master/.circleci/config.yml#L309

Basically, since the tests deploy Azure resources on our subscriptions, we'd like to be able to control when PRs are able to run the tests. So that would be either having a manual approval step that can only be approved by code owners, or only triggering the pipeline when a certain label is present on the PR (such as "ok-to-test"). Do you know if there are any existing ways to achieve this scenario with Azure DevOps builds and, if not, could you please consider this feature request?

**Comment by leantk** [comment]
@CecileRobertMichon is this issue resolved now with the comment triggers feature? If so I'll summarize the solution and close this out.

**Comment by CecileRobertMichon** [comment]
@leantk yes, it is! Thank you for your help :)

**Comment by leantk** [comment]
The solution for this issue was to edit the Pipeline definition to enforce comment triggers. This is a new feature under the Triggers tab in the Pipeline's editing experience. There is a checkbox under PR triggers that enforces all PR's only to be built after a comment is made by a team member on the PR. This can be enforced for all PR's or only PR's from non-team members.
